src/matchers/toBeArrayOfSize/index.js

```diff
@@ -0,0 +1,34 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = (actual, expected) => () => `${matcherHint('.not.toBeArrayOfSize', 'received', '')}
```

Comment:
If you change the matcher hint to just use the name of the matcher (negated in this case) like:
```js
matcherHint('.not.toBeArrayOfSize')
```

Under the hood this will default to `matcherHint('.not.toBeArrayOfSize', 'received', 'expected')` which wraps the expected section of the print out in a green color.